repos/backend/endpoints/files.js

```diff
@@ -10,10 +10,17 @@ const path = require('path')
 
 const saveFile = (app, config) => async (req, res) => {
   try {
-    const file = req.params.file
-    const content = req.params.content
-    const meta = await saveTestFile(config, file)
+    const path = req.body.path
```

Comment:
As discussed, we should try not to use 'path' as a variable name. How about 'location' ?

Reply:
updated